packages/salesforcedx-vscode-core/src/commands/forceDataSoqlQuery.ts

```diff
@@ -63,7 +63,7 @@ export class GetQueryAndApiInputs
         query = document.getText(editor.selection);
       }
     }
-    query = query!.replace('[', '').replace(']', '');
+    query = query!.replace('[', '').replace(']', '').replace(/(\r\n|\n)/g, " ");
```

Comment:
Thanks for making the change.
I think this will fail the linter that we have – did you run `npm run lint`?

We prefer to use single quotes and not double quotes. As you can see from the previous calls to `replace`.